styles/cartManagement.css

```diff
@@ -0,0 +1,76 @@
+.cart_page_title {
+  text-align: center;
+  margin: 4rem 0 5rem;
+}
+
+.cart_products_container {
+  width: 70%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2.5rem;
+}
+
+.cart_payment_receipt {
+  width: 30%;
+  height: fit-content;
+}
+
+.cart_card_resize {
+  width: 39rem;
+}
+
+.cart_card_content {
+  margin-top: 6px;
+}
```

Comment:
Use rem here since you are using rem everywhere, even for small sizes.

Reply:
okay , I will change

Reply:
Did the changes